### DISCUSSION TITLE
ubuntu-latest Apt repository list issues?

### DISCUSSION THREAD

**Discussion by ChadSikorra** [community/community#26211] [Actions]
<p>I have a script that gets kicked off during an action that installs some packages for a custom OpenLDAP configuration. Just today I started to get this error when running apt:</p>
<pre><code>Reading package lists...
W: Conflicting distribution: https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic InRelease (expected bionic but got stable)
</code></pre>
<p>Full error is here: <a href="https://github.com/FreeDSx/LDAP/runs/346718089" rel="nofollow noopener">https://github.com/FreeDSx/LDAP/runs/346718089</a></p>
<p>Seems like some kind of issue with the ubuntu-latest image?</p>

**Comment by morgo** [comment]
<p>I am <a href="https://github.com/vitessio/vitess/pull/5555/checks?check_run_id=346791776" rel="nofollow noopener">experiencing the same issue</a>.</p>

**Comment by 1000001101000** [comment]
<p>I am having the same problem</p>

**Comment by 1000001101000** [comment]
<p>I added “|| exit 0” to my apt-get update command which is allowing my workflow to work.</p>
<p>obviously, that won’t help if you depend on the particular repository that’s failing but it should help in some cases until the underlying issue is resolved.</p>

**Comment by Hanaasagi** [comment]
<p>Same problem for me.</p>

**Comment by jolestar** [comment]
<p>Same problem for me.</p>

**Comment by okunishinishi** [comment]
<p>Same problem for me.</p>

**Comment by royalpinto** [comment]
<p>Same issue :-\</p>

**Comment by akm** [comment]
<p>Same problem for me.</p>

**Comment by epatters** [comment]
<p>This looks like a GitHub bug. I was able to work around it by running this command before any apt-get commands.</p>
<pre><code>sudo rm -f /etc/apt/sources.list.d/dotnetdev.list /etc/apt/sources.list.d/microsoft-prod.list
</code></pre>
<p>For example, see this <a href="https://github.com/epatters/Catlab.jl/commit/3842b0b8a1e30818870d3adbe8e533c9cf44cf02" rel="nofollow noopener">commit</a>.</p>

**Comment by akm** [comment]
<p>I tried this again with <strong>–allow-releaseinfo-change</strong> and got the following message:</p>
<pre><code>Err:34 https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic/main amd64 Packages
  File has unexpected size (84552 != 156065). Mirror sync in progress? [IP: 40.76.35.62 443]
</code></pre>
<p>Microsoft repository might have another problem?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/8359adc6bac5f68a7b5f38e6ffe33890daac223a.png" data-download-href="/uploads/short-url/iJYGrByNcQSWLUiba7Og9aaHFlU.png?dl=1" title="スクリーンショット 2019-12-13 14.11.49 午後.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/18912/181096081-c604a661-e188-4376-a3f3-a7014a7cac3c.png" alt="スクリーンショット 2019-12-13 14.11.49 午後.png" data-base62-sha1="iJYGrByNcQSWLUiba7Og9aaHFlU" width="690" height="335" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/8359adc6bac5f68a7b5f38e6ffe33890daac223a_2_690x335.png, https://aws1.discourse-cdn.com/github/original/2X/8/8359adc6bac5f68a7b5f38e6ffe33890daac223a.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/8/8359adc6bac5f68a7b5f38e6ffe33890daac223a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/8359adc6bac5f68a7b5f38e6ffe33890daac223a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">スクリーンショット 2019-12-13 14.11.49 午後.png</span><span class="informations">995×484 110 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by traversaro** [comment]
<p>I guess that the core issue is in the dotnet’s apt repository that are serving buggy files, see <a href="https://github.com/dotnet/core/issues/3988" rel="nofollow noopener">https://github.com/dotnet/core/issues/3988</a> .</p>

**Comment by gustvn** [comment]
<p>insert this line before <strong>sudo apt update:</strong></p>
<p><strong>for apt_file in <code>grep -lr microsoft /etc/apt/sources.list.d/</code>; do sudo rm $apt_file; done</strong></p>

**Comment by ghost** [comment]
<p>Same here. Not a change that I expect to a CI system. :frowning:</p>

**Comment by pmatos** [comment]
<p>Same here - this seems to me to be the perfect example of why you should always run your jobs inside a well-defined container instead of inheriting whatever vm github puts you in.</p>
<p>Happy to be corrected on this matter though.</p>

**Comment by kaylangan** [comment]
<p>Thanks for the report and the patience! We’re investigating this issue and will post back when we have an update.</p>

**Comment by kaylangan** [comment, marked as answer]
<p>The recent changes that caused this issue have been reverted and individual feeds are being fixed so they can be used again. You can follow the latest status here: <a href="https://github.com/dotnet/core/issues/3995" rel="nofollow noopener">https://github.com/dotnet/core/issues/3995</a>. Thank you all again for the patience!</p>
